services/apps/alcs/src/providers/typeorm/migrations/1708556302044-cleanups.ts

```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class Cleanups1708556302044 implements MigrationInterface {
+  name = 'Cleanups1708556302044';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `DROP INDEX "alcs"."IDX_39c4f5ceb0f5a7a4c819d46a0d"`,
+    );
+    await queryRunner.query(
+      `COMMENT ON COLUMN "alcs"."notice_of_intent"."created_at" IS 'The date at which the noi was created in OATS'`,
```

Comment:
Interesting, so the comment was missing from the table?

Reply:
Yes, I kept getting this in my migration and I checked and it wasn't in a previous one. Maybe it was missed.